tests/snap/scala_spec.test.scala

```diff
@@ -239,4 +239,9 @@ object Annotations {
 
     @UserDefinedUpperCase def x
     @userDefinedLowerCase def y
-}
\ No newline at end of file
+}
+
+enum A:
+  case B, c, D
+
+enum A extends B derives X, Y, Z
```

Comment:
```suggestion
enum A extends B, C derives X, Y, Z
```
wouldn't we have the same problem with extends?

Reply:
Ah, right!